test/geometry_test.py

```diff
@@ -0,0 +1,211 @@
+import unittest
+
+from pygame import Vector2
+
+from geometry import Circle
```

Comment:
```suggestion
from pygame.geometry import Circle
```
This is consistent with how the other submodules are imported.